submissions/Nemooochka/html-css-popup/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <title>Html-css popup</title>
+
+    <script src="https://kit.fontawesome.com/559af75b05.js" crossorigin="anonymous"></script>
```

Comment:
It's also not a blocker, but try to avoid some external script for simple icon fonts. It can decrease the loading time of your page